src/main/java/org/isf/examination/service/ExaminationOperations.java

```diff
@@ -89,10 +92,15 @@ public PatientExamination getLastByPatID(int patID) throws OHServiceException	{
 
 	public List<PatientExamination> getLastNByPatID(int patID, int number) throws OHServiceException {
 		if (number > 0) {
-			return repository.findByPatient_CodeOrderByPexDateDesc(patID, PageRequest.of(0, number)).getContent();
+			return repository.findByPatient_CodeOrderByPexDateDesc(patID, PageRequest.of(0, number));
```

Comment:
With all these changes to methods it really would be good to add unit tests that verify they are doing what is expected, IMHO.

Reply:
okay your suggestion is to add test method for all methods that I added.... right?

Reply:
the difference at the the new method return paginated data with the page information. early we just return data without page info.